api-reference/10 UI Widgets/GridBase/1 Configuration/onToolbarPreparing.md

```diff
@@ -66,7 +66,7 @@ The following code shows how you can customize the toolbar using this function.
 ##### Angular
 
     <!--TypeScript-->
-    import { DxDataGridModule, DxButtonModule } from "devextreme-angular";
+    import { DxDataGridModule } from "devextreme-angular";
```

Comment:
Are you sure that the Button widget will be imported automatically once I add a button toolbar item?